onnxscript/optimizer/_constant_folding.py

```diff
@@ -959,9 +961,9 @@ class FoldConstantsPass(ir.passes.InPlacePass):
     def __init__(
         self,
         *,
-        shape_inference: bool,
-        input_size_limit: int,
-        output_size_limit: int,
+        shape_inference: bool = True,
+        input_size_limit: int = DEFAULT_CONSTANT_FOLD_INPUT_SIZE_LIMIT,
+        output_size_limit: int = DEFAULT_CONSTANT_FOLD_OUTPUT_SIZE_LIMIT,
```

Comment:
I wonder why you need two limits. Maybe we could set up the second one to None and replace None and the first value if it is still None.